packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json

```diff
@@ -150,13 +150,13 @@
   "hosting_dashboard_service_cdn_customer_has_cdn_v1_banner_msg_link_upgrade": "Migrer votre CDN",
   "hosting_dashboard_service_cdn_customer_has_cdn_v1_new_offers": "OVHcloud vous recommande les nouvelles offres CDN",
   "hosting_dashboard_service_web": "Sites web",
-  "hosting_dashboard_service_privatesql": "Base de données privée",
+  "hosting_dashboard_service_privatesql": "Web Cloud Databases",
```

Comment:
**questions:** Dont we need Translation for this? If not, better to maintain it as a constant.

Reply:
Hello, thanks for the review. "Web Cloud Databases" is the product name decided by marketing, and this product name is the same in all languages. 
This logic is the same for other products (CDN, SSL, SQL Server, OVHcloud...) and we don't have constants for these products.